surfman/src/platform/windows/angle/surface.rs

```diff
@@ -368,6 +386,23 @@ impl Device {
         })
     }
 
+    /// Given a D3D11 texture, create a surface texture that wraps that texture. This method is unsafe
+    /// in that the resulting surface is only valid on the current thread.
+    pub unsafe fn create_surface_texture_from_texture(
+        &mut self,
+        context: &mut Context,
+        size: &Size2D<i32>,
+        texture: *mut d3d11::ID3D11Texture2D
```

Comment:
Should this be a COM pointer?

Reply:
If not, document the reference counting semantics in the documentation.

Reply:
Looks like I can't make it a COM pointer, since the use case is openxr, and https://docs.rs/openxr/0.12.2/x86_64-pc-windows-msvc/openxr/d3d/enum.D3D11.html exposes `type SwapchainImage = *mut ID3D11Texture2D`. I'll add a comment saying that it is up to the caller to ensure that `texture` is kept live.

Reply:
You could still have made it a COM pointer, because you can wrap raw pointers in COM pointers.

I don't think it's a good idea to subvert the COM `IUnknown` reference counting semantics by requiring the caller to ensure liveness. `surfman` should be maintaining reference counts itself.

Reply:
TIL about `ComPtr::from_raw`. I'll submit a PR for that.